css/index.css

```diff
@@ -101,16 +165,66 @@ p {
     justify-content: space-evenly;
 }
 
-.middle-content .boxes .box {
+.middle-content .boxes  .box{
     width: 12.5%;
     height: 100px;
-    background: black;
+    /*background: black;*/
     margin: 20px 2.5%;
     color: white;
     display: flex;
     align-items: center;
     justify-content: center;
 }
+.box {
+    width: 12.5%;
+    height: 100px;
+   /* background: black;*/
+    margin: 20px 2.5%;
+    color: white;
+    display: flex;
+    align-items: center;
+    justify-content: center;}
+
+    .boxes .box1 {
+        background-color: teal;
```

Comment:
Interesting to see you opted for changing the HTML classes rather than using the `:nth-child(child-number)` to target the boxes. This is another case of multiple ways to skin a cat, and that is certainly one.